hmpps-sqs-spring-boot-autoconfigure/.editorconfig

```diff
@@ -2,9 +2,11 @@
 indent_size = 2
 ij_kotlin_allow_trailing_comma_on_call_site = true
 ij_kotlin_allow_trailing_comma = true
+ktlint_code_style = intellij_idea
```

Comment:
The new version of ktlint wanted to rewrite half the code, but this appears to make it much more lenient - because this was the code style in ktlint version <1.0 https://pinterest.github.io/ktlint/latest/rules/code-styles/